src/main/kotlin/no/nav/familie/klage/behandling/enhet/BehandlendeEnhetService.kt

```diff
@@ -0,0 +1,61 @@
+package no.nav.familie.klage.behandling.enhet
+
+import no.nav.familie.klage.behandling.BehandlingService
+import no.nav.familie.klage.behandlingshistorikk.BehandlingshistorikkService
+import no.nav.familie.klage.behandlingshistorikk.domain.HistorikkHendelse
+import no.nav.familie.klage.fagsak.FagsakService
+import no.nav.familie.klage.oppgave.OppgaveService
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import java.util.UUID
+
+@Service
+class BehandlendeEnhetService(
+    private val behandlingService: BehandlingService,
+    private val fagsakService: FagsakService,
+    private val behandlingshistorikkService: BehandlingshistorikkService,
+    private val oppgaveService: OppgaveService,
+) {
+    @Transactional
+    fun oppdaterBehandlendeEnhet(
```

Comment:
```suggestion
    fun oppdaterBehandlendeEnhetPåBehandling(
```

Nitpick, men kanskje noe sånt?
Av å lese metodenavnet bare så kan det virke ut som at man oppdaterer selve enheten.